apps/web/pages/api/integrations/office365video/add.ts

```diff
@@ -0,0 +1,33 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
We can remove this file and just keep the one in `packages/app-store/office365video/api/add.ts`